frontend/src/FullPageRoutes/RecordPage.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
```

Comment:
We probably want to make sure a user is logged in before they can record a video but I'm guessing for this PR, it's not required.